goforit.go

```diff
@@ -206,9 +206,12 @@ func DeletedCallback(cb evaluationCallback) Option {
 }
 
 type flagHolder struct {
-	flag         flags.Flag
-	clamp        clamp.Clamp
-	enabledCount atomic.Uint64
+	flag          flags.Flag
+	disabledCount atomic.Uint64
+	enabledCount  atomic.Uint64
+	clamp         clamp.Clamp
+	// pad out to end of 64-byte cacheline to prevent false sharing
```

Comment:
😬

Reply:
@vasi-stripe I revised this, and the struct is now only 32-bytes wide (with no padding) -- half the size of a cacheline on intel/amd/graviton hosts.  This ensures that it doesn't "straddle" a cacheline, which is good enough!